schema/add-assistive-tech.schema.js

```diff
@@ -73,20 +73,13 @@ const schema = Joi.object({
     }),
 
   issuesDiscovered: Joi.string()
-    .optional()
-    .allow(null, '')
+    .required()
     .custom((value, helpers) => maxWords(value, helpers, 250))
     .messages({
+      'any.required': 'Enter any issues discovered',
+      'string.empty': 'Enter any issues discovered',
```

Comment:
@helennickols You will likely want to alter these validation messages that are needed now the field is required.